src/components/routes/Host/ListingForm/ListingHelpText.tsx

```diff
@@ -0,0 +1,232 @@
+import * as React from 'react';
+import ListItem from 'shared/ListItem';
+import Svg from 'shared/Svg';
+import LazyImage from 'shared/LazyImage';
+
+interface ListingHelpTextInterfaceInterface {
+  [name: string]: React.ReactNode;
+}
+
+const HomeType = () => (
+  <div>
+    <h3>Type of Home</h3>
+    <h4>Entire place</h4>
+    <p>The whole space is dedicated to guests. This may be a house, apartment, condo, loft, etc.</p>
+    <h4>Private room</h4>
+    <p>A single room within a place is dedicated to guests. While other areas could be shared, guests have their own private room for sleeping.</p>
+    <h4>Shared room</h4>
+    <p>Bedrooms or a common area that could be shared with others are dedicated to guests.</p>
+  </div>
+);
+
+const Title = () => (
+  <div>
+    <h3>Listing Name</h3>
+    <p>&quot;Beautiful Home with Wifi and Views&quot;</p>
+  </div>
+);
+
+const Description = () => (
+  <div>
+    <h3>Listing Description</h3>
+    <p>Include a summary of your place to help guests imagine what it will be like to stay there. Note what makes your place special, mention what the neighborhood is like, and add anything else guests should know before arriving.</p>
+  </div>
+)
+
+const FullAddress = () => (
+  <div>
+    <h3>Full Address</h3>
+    <p>When adding the Full Address -- don’t forget to add a zip code!</p>
+  </div>
+);
+
+const ListingPicUrl = () => (
+  <div>
+    <h3>Cover Photo</h3>
+    <p>High-quality photos are a must as our guests are typically business travelers. Ideal image dimensions are 1440x960. The maximum file size is 5 MB and the file types we accept are .jpg &amp; .png.</p>
+    <h4>Examples:</h4>
+    <div className="images-container">
+      <LazyImage className="image-container--horizontal" src="https://static.beenest.com/images/photo-examples/Do_LA_Bedroom.jpg" />
+      <LazyImage className="image-container--horizontal" src="https://static.beenest.com/images/photo-examples/Do_LA_Bedroom_2.jpg" />
+    </div>
+  </div>
+);
+
+const Photos = () => (
+  <div>
+    <h3>Listing Photos</h3>
+    <p>Beenest recommends listings have at least 5 high-definition photos available. Photos are the best way to show off your place to potential guests. <span>Pro tip:</span> provide multiple well-lit photos with different angles of bedrooms, bathrooms, and public spaces so guests know exactly what to expect.</p>
+    <h4>Do:</h4>
+    <ul>
+      <ListItem noHover prefixColor="correct" start="tiny">
+        <Svg className="prefix" src="utils/check-circle" />
+        <span>Take horizontal photos</span>
+      </ListItem>
+      <ListItem noHover prefixColor="correct" start="tiny">
+        <Svg className="prefix" src="utils/check-circle" />
+        <span>Take photos when natural light is brightest</span>
+      </ListItem>
+      <ListItem noHover prefixColor="correct" start="tiny">
+        <Svg className="prefix" src="utils/check-circle" />
+        <span>Do provide multiple angles</span>
```

Comment:
Since this is preceded with a "Do:" (the `h4` above) I'd just say "Provide multiple angles" here